cluster-provision/ocp/4.3/publish.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-docker tag kubevirtci/ocp-4.3-provision:latest docker-registry.upshift.redhat.com/kubevirtci/ocp-4.3:latest
-docker push docker-registry.upshift.redhat.com/kubevirtci/ocp-4.3:latest
+docker tag kubevirtci/ocp-4.3-provision:latest quay.io/kubevirtci/ocp-4.3:latest
+docker push kubevirtci/ocp-4.3-provision:latest
```

Comment:
I think that you need to specify the version that you tagged here `quay.io/kubevirtci/ocp-4.3:latest`, otherwise the tag is redundant...